### PR TITLE
Status update(s)

### PR DESCRIPTION
**Pull request by anodcire**
https://news.vice.com/story/trump-signed-a-stack-of-executive-orders-moments-after-he-became-president

### CODE REVIEW

File: _data/data.yaml [hunk 1]
```diff
@@ -1458,7 +1458,7 @@ tabs:
             source: 'http://www.nbcnews.com/politics/2016-election/full-list-donald-trump-s-rapidly-changing-policy-positions-n547801'
           -
             text: 'Defund Planned Parenthood.'
-            status: notStarted
+            status: Started
```
Comment (alanfriedman) [inline _data/data.yaml:1461]: I can't find any reports of him starting this.

File: _data/data.yaml [hunk 2]
```diff
@@ -64,7 +64,7 @@ tabs:
             source: 'https://www.donaldjtrump.com/press-releases/donald-j.-trump-delivers-groundbreaking-contract-for-the-american-vote1'
           -
             text: 'Donald Trump will refuse the $400,000 annual president''s salary or only take $1'
-            status: notStarted
+            status: Complete
```
Comment (alanfriedman) [inline _data/data.yaml:67]: I can't find any reports of him starting this.

**Review by alanfriedman — CHANGES_REQUESTED** [review]

File: _data/data.yaml (hunk 1 above)
Comment (anodcire) [reply _data/data.yaml:1461]: http://www.politifact.com/truth-o-meter/promises/trumpometer/promise/1357/defund-planned-parenthood/

It's in the works for this Sunday.

File: _data/data.yaml (hunk 2 above)
Comment (anodcire) [reply _data/data.yaml:67]: You're correct, it seems as though it hasn't been confirmed. Although it seems he may end up taking $1 for his salary.

Comment (alanfriedman) [reply _data/data.yaml:67]: Ok let's keep this as notStarted until progress is made.

File: _data/data.yaml (hunk 1 above)
Comment (alanfriedman) [reply _data/data.yaml:1461]: Ok thanks

**Comment by MrLuit** [comment]
The status for started is 'inProgress' not 'Started' :)

Also, it's 'achieved' not 'Complete' 



notStarted, inProgress, broken, achieved

**Comment by VirenMohindra** [comment]
This is now deprecated, if you'd like to contribute, please create a new PR with the new `data.yaml`.

Thank you!

**Pull request closed without merging**